internal/gamepad/gamepad_darwin.go

```diff
@@ -73,82 +41,80 @@ func newNativeGamepadsImpl() nativeGamepads {
 }
 
 func (g *nativeGamepadsImpl) init(gamepads *gamepads) error {
-	var dicts []C.CFDictionaryRef
+	var dicts []_CFDictionaryRef
 
-	page := C.kHIDPage_GenericDesktop
+	page := kHIDPage_GenericDesktop
 	for _, usage := range []uint{
-		C.kHIDUsage_GD_Joystick,
-		C.kHIDUsage_GD_GamePad,
-		C.kHIDUsage_GD_MultiAxisController,
+		kHIDUsage_GD_Joystick,
+		kHIDUsage_GD_GamePad,
+		kHIDUsage_GD_MultiAxisController,
 	} {
-		pageRef := C.CFNumberCreate(C.kCFAllocatorDefault, C.kCFNumberIntType, unsafe.Pointer(&page))
+		pageRef := _CFNumberCreate(kCFAllocatorDefault, kCFNumberIntType, unsafe.Pointer(&page))
 		if pageRef == 0 {
 			return errors.New("gamepad: CFNumberCreate returned nil")
 		}
-		defer C.CFRelease(C.CFTypeRef(pageRef))
+		defer _CFRelease(_CFTypeRef(pageRef))
 
-		usageRef := C.CFNumberCreate(C.kCFAllocatorDefault, C.kCFNumberIntType, unsafe.Pointer(&usage))
+		usageRef := _CFNumberCreate(kCFAllocatorDefault, kCFNumberIntType, unsafe.Pointer(&usage))
 		if usageRef == 0 {
 			return errors.New("gamepad: CFNumberCreate returned nil")
 		}
-		defer C.CFRelease(C.CFTypeRef(usageRef))
+		defer _CFRelease(_CFTypeRef(usageRef))
 
-		keys := []C.CFStringRef{
-			C.cfStringRefIOHIDDeviceUsagePageKey(),
-			C.cfStringRefIOHIDDeviceUsageKey(),
+		keys := []_CFStringRef{
+			_CFStringCreateWithCString(kCFAllocatorDefault, kIOHIDDeviceUsagePageKey, kCFStringEncodingUTF8),
+			_CFStringCreateWithCString(kCFAllocatorDefault, kIOHIDDeviceUsageKey, kCFStringEncodingUTF8),
 		}
-		values := []C.CFNumberRef{
+		values := []_CFNumberRef{
 			pageRef,
 			usageRef,
 		}
 
-		dict := C.CFDictionaryCreate(C.kCFAllocatorDefault,
+		dict := _CFDictionaryCreate(kCFAllocatorDefault,
 			(*unsafe.Pointer)(unsafe.Pointer(&keys[0])),
 			(*unsafe.Pointer)(unsafe.Pointer(&values[0])),
-			C.CFIndex(len(keys)), &C.kCFTypeDictionaryKeyCallBacks, &C.kCFTypeDictionaryValueCallBacks)
+			_CFIndex(len(keys)), *(**_CFDictionaryKeyCallBacks)(unsafe.Pointer(&kCFTypeDictionaryKeyCallBacks)), *(**_CFDictionaryValueCallBacks)(unsafe.Pointer(&kCFTypeDictionaryValueCallBacks)))
```

Comment:
Hmm, the pointer indirection seems a dirty hack to trick go-vet. I'd skip go-vet for this package if we need. Is there a option to skip a specific package for go-vet by the way?

Reply:
I don't think there is a way. I feel it's better to use the hack for the 6 or so needs of it and still have go get for the rest of the package then to try and not have go get at all

Reply:
You mean go vet?

Reply:
Yes

Reply:
Sure, there are pros and cons, and keeping go-vet enabled seems slightly better.